api/db/database-builder/factory/build-user.js

```diff
@@ -16,7 +16,7 @@ const buildUser = function buildUser({
   lastName = faker.name.lastName(),
   email,
   username = firstName + '.' + lastName + faker.random.number({ min: 1000, max: 9999 }),
-  password,
+  password = 'pix123',
```

Comment:
```suggestion
  password = 'Password123',
```
Pour respecter le format demandé.